CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ if(POLICY CMP0135)
     cmake_policy(SET CMP0135 NEW) # valid for DOWNLOAD_EXTRACT_TIMESTAMP option in CMake 3.24 and later
 endif()
 
+project(libavif LANGUAGES C VERSION 1.1.0)
+
```

Comment:
[CMake documentation on `project()`](https://cmake.org/cmake/help/latest/command/project.html#usage) says:

> **Note:**  Call the `project()` command near the top of the top-level `CMakeLists.txt`, but after calling [cmake_minimum_required()](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html#command:cmake_minimum_required).

But I did not find guidance in CMake documentation on whether `project()` should be called before or after `cmake_policy()`. In this pull request I preserve the current order of calling `project()` after `cmake_policy()`.

Reply:
Yes, [CMake documentation on cmake_minimum_required()](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html) says it should be the first call in the top-level CMakeLists.txt file:

> **Note:**  Call the `cmake_minimum_required()` command at the beginning of the top-level `CMakeLists.txt` file even before calling the [project()](https://cmake.org/cmake/help/latest/command/project.html#command:project) command.

Do you recommend we call `project()` before `make_policy()`?

Reply:
Thinking about it a little, I think it would only matter if one of the policies affected `project()` and you wanted to keep the grouping. So think the order you have makes sense.